app/javascript/app/components/data-explorer-content/api-documentation/api-documentation.js

```diff
@@ -0,0 +1,217 @@
+import { createElement, PureComponent } from 'react';
+import { PropTypes } from 'prop-types';
+import Component from './api-documentation-component';
+
+const { CW_API } = process.env;
+const { ESP_API } = process.env;
+
+const API_CALLS = {
+  'historical-emissions': [
+    {
+      title: 'GET /api/v1/data/historical_emissions/meta',
+      url: `${CW_API}/data/historical_emissions/meta`,
+      description:
+        'Returns a Link header with meta endpoint urls for discovery (can be used with a HEAD request)',
+      extra: 'This response only has headers not body'
+    },
+    {
+      title: 'GET /api/v1/data/historical_emissions',
+      url: `${CW_API}/data/historical_emissions`,
+      description: 'Retrieves time series data for historical emissions',
+      queryParams: [
+        {
+          name: 'data_sources',
+          parameter: 'source_ids[]',
+          description: 'emission data source id (CAIT, PIK, UNFCCC)'
+        },
+        {
+          name: 'gwps',
+          parameter: 'gwp_ids[]',
+          description: 'emission gwps id'
+        },
+        {
+          name: 'gases',
+          parameter: 'gas_ids[]',
+          description: 'emission data source id (CAIT, PIK, UNFCCC)'
+        },
+        {
+          name: 'sectors',
+          parameter: 'sector_ids[]',
+          description: 'sector id'
+        },
+        {
+          name: 'regions',
+          parameter: 'regions[]',
+          description: 'region ISO code 3'
+        },
+        {
+          name: 'start_year',
+          parameter: 'start_year',
+          description: 'Show results from this year onwards'
+        },
+        {
+          name: 'end_year',
+          parameter: 'end_year',
+          description: 'Show results up to this year'
+        }
+      ],
+      extra:
+        'All this parameters accept multiple values, separated by commas. Response is paginated. Pagination headers are in place.'
+    }
+  ],
+  'emission-pathways': [
+    {
+      title: 'GET /api/v1/data/emission_pathways/meta',
+      url: `${ESP_API}/data/emission_pathways/meta`,
```

Comment:
Emissions pathways meta link is triggering an error . 

![image](https://user-images.githubusercontent.com/6906348/42682764-be28d1b4-868b-11e8-8457-62660ff15495.png)